File: src/server/chat/src/main/java/smilegate/plop/chat/dto/LastMessage.java
package smilegate.plop.chat.dto;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class LastMessage {
    private String message_id;
    private String sender_id;
    private String content;
    private LocalDateTime created_at;
}

File: src/mobile/android/app/src/main/java/com/plop/plopmessenger/data/dto/response/LastMessage.kt
package com.plop.plopmessenger.data.dto.response


import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class LastMessage(
    val content: String,
    @SerializedName("created_at")
    val createdAt: LocalDateTime,
    @SerializedName("message_id")
    val messageId: String,
    @SerializedName("sender_id")
    val senderId: String
)